start/sw/web-app/django.md

```diff
@@ -10,130 +10,103 @@ Django is a high-level Python Web framework that encourages rapid development an
 Django is one of the most popular Python Web Frameworks. It has very useful web page. Visit it by clicking on [this link](https://www.djangoproject.com/). 
 Because you will need to work with Python packages, write this command to your command line.
 
-For Python:
-
-```
-sudo dnf install python-pip
-```
-
-or Python 3:
-
 ```
-sudo dnf install python3-pip
+$ sudo dnf install python3-pip
 ```
 
-# Get started
+## Get started
 
-## How to install virtualenv
+### How to install virtualenv
 
 It's recommended to keep your project inside virtual environment. But you can skip this step if you want. 
 Enter this command to your command line for installation:
 
 ```
-sudo dnf install python-virtualenv
-```
-
-or installation for Python3:
-
-```
-sudo dnf install python3-virtualenv
+$ sudo dnf install python3-virtualenv
 ```
 
 
 Get inside your working directory or create new where you want to store your virtual environment:
 
 ```
-mkdir working-dir
-cd working-dir
+$ mkdir working-dir
+$ cd working-dir
```

Comment:
These steps are missing from the [python page](https://developer.fedoraproject.org/tech/languages/python/python-installation.html). Could you move it over there?